snowplow/iglu-client-embedded/schemas/com.metabase/embed_flow/jsonschema/1-0-3

```diff
@@ -0,0 +1,222 @@
+          "minimum": 0,
+          "maximum": 2147483647
+        },
+        "disabled": {
+          "type": "integer",
+          "description": "Number of disabled parameters in the artifact.",
+          "minimum": 0,
+          "maximum": 2147483647
+        }
+      }
+    },
+    "first_published_at": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "format": "date-time",
+      "description": "The timestamp when the artifact was first published."
+    },
+    "language": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "The backend or view language of the artifact.",
+      "maxLength": 1024
+    },
+    "location": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "enum": [
+        "code_overview",
+        "code_params",
+        "code_appearance"
+      ],
+      "description": "The location in the code where the event is triggered."
+    },
+    "code": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "enum": [
+        "backend",
+        "view"
+      ],
+      "description": "Type of the language of the artifact where the event is triggered."
+    },
+    "appearance": {
+      "type": [
+        "object",
+        "null"
+      ],
+      "description": "The appearance settings of the artifact.",
+      "properties": {
+        "title": {
+          "type": "boolean",
+          "description": "Indicates if the title is visible."
+        },
+        "border": {
+          "type": "boolean",
+          "description": "Indicates if the border is visible."
+        },
+        "theme": {
+          "type": [
+            "string"
+          ],
+          "enum": [
+            "light",
+            "night",
+            "transparent"
+          ],
+          "description": "The theme of the artifact's appearance."
+        },
+        "font": {
+          "type": "string",
+          "enum": [
+            "instance",
+            "custom"
+          ],
+          "description": "The font type used in the artifact."
+        },
+        "hide_download_button": {
+          "type": [
+            "boolean",
+            "null"
+          ],
+          "description": "Indicates if the download button is hidden. It will be null on OSS instance since it's not supported."
+        },
+        "downloads": {
+          "type": [
+            "object",
+            "null"
+          ],
+          "description": "Indicates which download types are enabled, it will be null on OSS as they can't disable them",
+          "properties": {
+            "pdf": {
+              "type": "boolean",
+              "description": "Indicates if PDF exports are enabled. Only applicable to dashboards."
+            },
+            "dashcard": {
+              "type": "boolean",
+              "description": "Indicates if dashboard card downloads are enabled."
+            }
+          }
+        }
+      }
```

Comment:
This is the only change in this snowplow file. Everything else is copied from 1-0-2

Reply:
https://www.notion.so/metabase/Snowplow-integration-5da1f874beda4153b4fccfa6c1e77caa?pvs=4#4db90a8dfc1640b087d9a4e5e95c6576

Looks like type change is a breaking change. At least from that doc. That's a good call.

Reply:
Great call @npretto! I never worked with Snowplow before so I didn't know that Snowplow follows semver, my bad. I'll update the schema to be backwards-compatible 👍🏻